### ISSUE TITLE
Unit test failures when pandas is not installed

### ISSUE THREAD

**Issue by jlstevens** [ioam/holoviews#1459]
Running:

```
nosetests  -a  '!optional' testcomparisondimension.py
```

Results in errors such as 
```
======================================================================
ERROR: test_dimension_comparison_cyclic_unequal (tests.testcomparisondimension.DimensionsComparisonTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/jstevens/Desktop/development/holoviews/tests/testcomparisondimension.py", line 21, in setUp
    self.dimension8 = Dimension('dim1', values=['a', 'b'])
  File "/Users/jstevens/Desktop/development/holoviews/holoviews/core/dimension.py", line 317, in __init__
    all_params['values'] = list(unique_array(values))
  File "/Users/jstevens/Desktop/development/holoviews/holoviews/core/util.py", line 770, in unique_array
    return arr[np.sort(uniq_inds)]
TypeError: only integer scalar arrays can be converted to a scalar index
```

I think this is a bug that shows up when pandas isn't installed.

**Comment by philippjfr** [comment]
I think this has been fixed, just double check before closing.

**Comment by jlstevens** [comment]
Should be fixed since #1420. Closing.
